### ISSUE TITLE
Bug in compile script

### ISSUE THREAD

**Issue by krystalgamer** [Jessecar96/SteamBot#707]
So i spent the last half hour trying to get what was the problem while compiling in ubuntu SteamBot.
And the problem was that in the portuguese OS the desktop is called "Área de Trabalho" and when the compile script tries to access it fails because of the space.

Here's the error i have:

/home/nams/Área de Trabalho/bot/SteamBot/.nuget/NuGet.targets: error : Command 'bash /home/nams/Área de Trabalho/bot/SteamBot/.nuget/../.ci/exec-with-retry.sh mono --runtime=v4.0.30319 /home/nams/Área de Trabalho/bot/SteamBot/.nuget/NuGet.exe install "packages.config" -source ""  -RequireConsent -solutionDir "/home/nams/Área de Trabalho/bot/SteamBot/"' exited with code: 127.

Basicly it stops here -> /home/nams/Área , the correct way would be /home/nams/"Área de Trabalho"/
or "/home/nams/"Área de Trabalho/"

tl dr Folders with spaces break the script making impossible to compile

Solution: compile the bot in a folder with no spaces

**Comment by BlueRaja** [comment]
We have a compile script?

**Comment by krystalgamer** [comment]
So what you call these? https://github.com/Jessecar96/SteamBot/tree/master/.ci

**Comment by BrokenSilence** [comment]
Translate your windows to english.xD

2015-02-01 22:39 GMT+00:00 krystalgamer <notifications@github.com>:

**Comment by WildCard65** [comment]
@BrokenSilence Only problem: It's not Windows, it's Ubuntu, a linux OS

**Comment by krystalgamer** [comment]
Btw this isn't only steambot related. I had trouble with some projects while ago and the problem was the same.

**Comment by Jessecar96** [comment]
The .ci folder is for Travis CI, not fo building it yourself.

**Comment by krystalgamer** [comment]
I wasn't building it myself. I didn't mention this in first place but this happened first with MonoDevelop.
The problem looks like it isn't Travis CI.
The real problem is Nuget.targets: https://github.com/Jessecar96/SteamBot/blob/cdea4e66419d184fe6ae73a1c9d2513090626398/.nuget/NuGet.targets#L57

This line particularly.

**Comment by nicolas-martin** [comment]
@krystalgamer I'm having the same problem using osx and xamarin. But I don't have spaces in my path. Were you able to fix it?

**Comment by BlueRaja** [comment]
@nicolas-martin https://github.com/Jessecar96/SteamBot/pull/710
